commit ac155d48e9474aa4586c69f5a8d040997596691f (refs/changes/54/2754/1)
Author: David Pacheco <dap@joyent.com>
Date:   2017-10-09T11:37:41-07:00 (2 years ago)
    
    MANTA-3459 manta-adm manual page could use example of listing compute ids and storage ids

diff --git a/docs/man/man1/manta-adm.md b/docs/man/man1/manta-adm.md
index 4231665..b1b9927 100644
--- a/docs/man/man1/manta-adm.md
+++ b/docs/man/man1/manta-adm.md
@@ -471,6 +471,10 @@ Example: list hostnames in form suitable for "sdc-oneachnode -n":
 
     # manta-adm cn -n
 
+Example: list storage CNs with their associated storage id (used in object
+metadata) and compute ids (used by the compute jobs subsystem):
+
+    # manta-adm cn -o host,admin_ip,compute_id,storage_ids storage
 
 ### "genconfig" subcommand
 
diff --git a/man/man1/manta-adm.1 b/man/man1/manta-adm.1
index acd8e0d..768304d 100644
--- a/man/man1/manta-adm.1
+++ b/man/man1/manta-adm.1
@@ -506,6 +506,15 @@ Example: list hostnames in form suitable for "sdc\-oneachnode \-n":
 # manta\-adm cn \-n
 .fi
 .RE
+.PP
+Example: list storage CNs with their associated storage id (used in object
+metadata) and compute ids (used by the compute jobs subsystem):
+.PP
+.RS
+.nf
+# manta\-adm cn \-o host,admin_ip,compute_id,storage_ids storage
+.fi
+.RE
 .SS "genconfig" subcommand
 .PP
 \fB\fCmanta\-adm genconfig "lab" | "coal"\fR
